nixos/tests/all-tests.nix

```diff
@@ -91,6 +91,7 @@ in
   flannel = handleTestOn ["x86_64-linux"] ./flannel.nix {};
   fluentd = handleTest ./fluentd.nix {};
   fontconfig-default-fonts = handleTest ./fontconfig-default-fonts.nix {};
+  freeswitch = handleTest freeswitch.nix {};
```

Comment:
Oh, mistake here, should be `./freeswitch.nix`, that's why ofborg couldn't run the test.

Reply:
Fixed!